api/build.go

```diff
@@ -407,6 +407,16 @@ func skipEmptyBuild(p *pipeline.Build) string {
 //   type: integer
 //   maximum: 100
 //   default: 10
+// - in: query
+//   name: before
+//   description: filter builds before a certain time
+//   type: integer
+//   default: 1
+// - in: query
+//   name: after
+//   description: filter builds after a certain time
```

Comment:
Do you think it's worth clarifying this is for builds `created`?

```suggestion
//   description: filter builds created after a certain time
```